mailer.rst

```diff
@@ -1489,7 +1491,7 @@ FailedMessageEvent
     {
         // e.g you can get more information on this error when sending an email
         $event->getError();
-        
+
         // do something with the message
     }
 
```

Comment:
Same here for the Embedding Images section, ended with a deprecated directive for version 6.2

Reply:
Done in 7f00524ba6d1e6562199e710ff8b32d304f4bbce